### ISSUE TITLE
Question about "membrane docking"

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#18464]
Definition | Parents
-- | --
The initial attachment of a membrane or protein to a target membrane. Docking requires only that the proteins come close enough to interact and adhere.

Should this term, and it's chidren really  be MF "membrane adaptor" terms?

**Comment by ukemi** [comment]
I don't think so. Docking is the process where the molecules interact so that something can adhere to a membrane.   See figure 1 in the reference for this term. PMID:27875684 

The protein membrane adaptor term seems to be for something that is more generally used to hold a protein at a specific location in a membrane, like in membrane structuring or creating polarity.

'Interacting selectively and non-covalently with both a protein or protein complex and a membrane, in order to maintain the localization of the protein at a specific membrane location.'

**Comment by ValWood** [comment]
It still looks like a molecular function" rather than a process:
For refernce from PMID:27875684

<img width="533" alt="Screenshot 2019-12-20 at 10 44 26" src="https://user-images.githubusercontent.com/7359272/71249932-414c6600-2316-11ea-8ef3-d913bd39ca64.png">


And the membrane anchors don't only anchor other proteins, many attach organelles to the membrane (cytokinetic ring etc).


the children of membrane docking are:

GO:0022615    protein to membrane docking | is_a
-- | --
GO:0022614    membrane to membrane docking | is_a
GO:0140056    organelle localization by membrane tethering | is_a

and under  GO:0140056    organelle localization by membrane tethering | is_a
GO:0048278    vesicle docking | is_a
-- | --
GO:0061817    endoplasmic reticulum-plasma membrane tethering | is_a
GO:1990854    vacuole-ER tethering | is_a
GO:0010381    peroxisome-chloroplast membrane tethering | is_a
GO:0140057    vacuole-mitochondria membrane tethering | is_a
GO:0071562    nucleus-vacuole junction assembly | is_a
GO:0097711    ciliary basal body-plasma membrane docking | is_a
GO:0062095    endoplasmic reticulum-peroxisome tethering | is_a
GO:0016240    autophagosome membrane docking | is_a
GO:1990456    mitochondrion-endoplasmic reticulum membrane tethering | is_a


All of which seem to be describing the activity of a "teher" except
GO:0071562    nucleus-vacuole junction assembly | is_a
but this paper seems to be describing a single protein-protein interaction so "junction assembly" 
isn't really a process here.

**Comment by ValWood** [comment]
The things I have annotated to membraner docking are these:
https://www.pombase.org/term/GO:0048278
SNAREs etc. This seems correct for "docking" (ALthough I still think it is an activity.

I think possibly, the problem is that the parent
is saying something different

Definition | Parents
-- | --
The process by which an organelle membrane interacts with another membrane via molecular tethers that physically bridge the two membranes and attach them to each other.

so maybe  we need to figure out  what the parts of the process are, and which parts represent "activities"

**Comment by ValWood** [comment]
Definitely some duplication, it seems:


9 | vesicle docking involved in exocytosis (GO:0006904) | ...
-- | -- | --
  | 8 | vesicle tethering involved in exocytosis (GO:0090522) | ...
  | 2 | (vesicle docking involved in exocytosis (GO:0006904) AND vesicle tethering involved in exocytosis (GO:0090522))

**Comment by ValWood** [comment]
Out of date. I'll make a new ticket for this as part of "vesicle-mediated transport" review
